release/gcb_build_lib.sh

```diff
@@ -44,6 +44,10 @@ source "${SCRIPTPATH}/json_parse_shared.sh"
 # }
 #
 
+BUILD_FAILED=0
+# XXX this is ugly, but BUILD_FAILED is being used by calling scripts which call run_build
```

Comment:
Interesting. In the previous state, it didn't have an export. I didn't variables could be used in other scripts without `export`.